crates/cli-support/src/js/mod.rs

```diff
@@ -322,12 +322,33 @@ impl<'a> Context<'a> {
         // Deno removed support for .wasm imports in https://github.com/denoland/deno/pull/5135
         // the issue for bringing it back is https://github.com/denoland/deno/issues/5609.
         format!(
-            "const file = new URL(import.meta.url).pathname;
-            const wasmFile = file.substring(0, file.lastIndexOf(Deno.build.os === 'windows' ? '\\\\' : '/') + 1) + '{}_bg.wasm';
-            const wasmModule = new WebAssembly.Module(Deno.readFileSync(wasmFile));
-            const wasmInstance = new WebAssembly.Instance(wasmModule, imports);
+            "const url = new URL(import.meta.url);
+
+            let wasmCode = '';
+            switch (url.protocol) {{
+                case 'file:':
+                    let file = new URL(import.meta.url).pathname;
```

Comment:
You already have `url` instance from above:

```suggestion
                    let file = url.pathname;
```